BookPlayer/Library/ItemListViewController.swift

```diff
@@ -22,6 +22,11 @@ class ItemListViewController: UIViewController, ItemList, ItemListAlerts, ItemLi
 
     @IBOutlet weak var tableView: UITableView!
 
+    private var previousLeftButtons: [UIBarButtonItem]?
+    lazy var selectButton: UIBarButtonItem = {
```

Comment:
also, not sure if the braces are necessary? isn't it enough to declare it lazy and have a normal assign? 🤔 haven't read the docs about this in a long time tbh